guides/source/action_view_overview.md

```diff
@@ -42,47 +64,119 @@ $ bin/rails generate scaffold article
       [...]
 ```
 
-There is a naming convention for views in Rails. Typically, the views share their name with the associated controller action, as you can see above.
-For example, the index controller action of the `articles_controller.rb` will use the `index.html.erb` view file in the `app/views/articles` directory.
-The complete HTML returned to the client is composed of a combination of this ERB file, a layout template that wraps it, and all the partials that the view may reference. Within this guide, you will find more detailed documentation about each of these three components.
+The file names follow a Rails naming convention. They share their name with the
+associated controller action. For example the `index.html.erb`, `edit.html.erb`,
+etc.
 
-As mentioned, the final HTML output is a composition of three Rails elements: `Templates`, `Partials` and `Layouts`.
-Below is a brief overview of each of them.
+By following this naming convention, Rails will automatically find and render
+the matching view at the end of a controller action, without you having to
+specify it. For example, the `index` action in the `articles_controller.rb` will
+automatically render the `index.html.erb` view inside the `app/views/articles/`
+directory. The name and the location of the file are both important.
+
+The final HTML returned to the client is composed of a combination of the
+`.html.erb` ERB file, a layout template that wraps it, and all the partials that
+the ERB file may reference. In the rest of this guide, you will find more
+details about each of the three components: `Templates`, `Partials`, `Layouts`.
 
 Templates
 ---------
 
-Action View templates can be written in several ways. If the template file has a `.erb` extension then it uses a mixture of ERB (Embedded Ruby) and HTML. If the template file has a `.builder` extension then the `Builder::XmlMarkup` library is used.
+Action View templates can be written in different formats. If the template file
+has a `.erb` extension, it uses embedded Ruby to build an HTML response. If the
+template has a `.jbuilder` extension, it uses the
+[Jbuilder](https://github.com/rails/jbuilder) gem to build a JSON response. And
+a template with a `.builder` extension uses the
+[`Builder::XmlMarkup`](https://github.com/jimweirich/builder) library to build
```

Comment:
This is a bit unrelated to this PR, but I wonder if we are really using Jim Weirich's version of this Gem. His repo hasn't been updated in 11 years (for obvious reasons), but [RubyGems says](https://rubygems.org/gems/builder/versions/3.2.4) the latest version is 3.2.4. Is this gem being maintained elsewhere?

Reply:
It seems it's being maintained by Aaron: https://github.com/tenderlove/builder who is also mentioned as an [owner](https://rubygems.org/gems/builder) on Rubygems.

Reply:
Should we link to that repo instead? I'm confused as to why RubyGems points to Jim's repo. Aaron's repo seem to have the right info on gemspec.

Reply:
Should this even be mentioned in the official guides? Since it's not really a gem maintained by the Rails team as such.

Reply:
The homepage of tenderlove's version points to tenderlove's version.
https://github.com/tenderlove/builder/blob/b3941919becdce4021a3f70838240442f155f268/builder.gemspec#L43
I'd expect the Rubygems to link to that version as well.

Reply:
@p8 yeah. This is quite confusing. I wonder if there's some caching happening in that page?

Reply:
@MatheusRich I think it needs a new release. The metadata was updated in a unreleased version.

Reply:
Closing the loop here, this is now living under [rails/builder](https://github.com/rails/builder): https://github.com/rails/rails/commit/d462fb54b459e222efc2e2480397af44a9c55361